Reformat and update docs (#66)
primer/presentations: content/presentation-formats/figma.mdx

---
title: Figma
---

The GitHub presentation template is available as a Figma library for members of the GitHub organization.

### How to use

> To use:

Reviewer: ```suggestion
```

> 1. Open `GitHub Presentation Template` located in the Primer team in Figma.

Reviewer: ```suggestion
1. Open `GitHub Presentation Template` located in the Primer team in Figma
```

<img
  width="656"
  alt="Presentation template in Figma"
  src="https://user-images.githubusercontent.com/10384315/101224794-96c37500-3644-11eb-9bf6-b4b38c7667eb.png"
/>

2. Duplicate the file into your own drafts

<img
  width="656"
  alt="doctocat-figma-duplicate"
  src="https://user-images.githubusercontent.com/10384315/101224924-ec981d00-3644-11eb-8b6e-7a3aa0fba6a2.png"
/>

3. Open the duplicated file and configure your own presentation

> <Note variant="warning">Tip: Don't forget to rename the duplicated presentation file.</Note>

Reviewer: ```suggestion
<Note variant="warning">**Tip:** Remember to rename the duplicated presentation file.</Note>
```
